docs/netstandard-20/packaging.md

```diff
@@ -0,0 +1,350 @@
+        ├───System.Threading.Tasks_4.0.11
+        ├───System.Threading.Timer_4.0.1
+        ├───System.Threading_4.0.11
+        ├───System.Xml.ReaderWriter_4.0.11
+        ├───...
+        └───System.Xml.XDocument_4.0.11
+
+Since we want to decouple .NET Standard from .NET Core, we decided to go with a
+different assembly set for .NET Standard 2.0, namely a monolithic assembly
+called `netstandard.dll` which contains the entirety of .NET Standard.
+
+In order to stay compatible with .NET Standard 1.x, the fine grained assemblies
+need to type forward to `netstandard.dll` when they are consumed from .NET
+Standard 2.0. Logically, this requires the following steps:
+
+1. .NET Standard 2.0 has newer versions of the fine grained assemblies that
+   type forward to `netstandard.dll`
+
+2. When consuming .NET Standard 1.x binaries from 2.0, references to the fine
+   grained assemblies are unified with the higher versions in .NET Standard 2.0
+   so that the type forwarding versions are used.
+
+We could use NuGet to model this:
+
+1. We'd have to ship updates to all the fine grained packages to incude a type
+   forwarding facade for .NET Standard 2.0.
+
+2. To ensure these higher versions are used, the package for .NET Standard 2.0
+   has to continue to depend on the fine grained packages so that it can express
+   the dependency on the higher version that has the type forwarding facade.
+
+However, we'd like to stop using these fine grained packages for both, .NET
+Standard as well as .NET Core:
+
+* **.NET Standard**. Since .NET Standard 2.0 no longer uses the fine grained
+  factoring, we don't want to continue to depend on the fine-grained
+  `System.Runtime` based packages.
+
+* **.NET Core**. While the `System.Runtime` based factoring will continue to be
+  used for .NET Core we don't expect .NET Core to ship these assemblies as
+  indiviudal packages either. Instead, we'd like to follow .NET Standard and
+  ship a monolithic package that contains most of the .NET Core assembly set.
+
+In order to solve this problem, we'll instead:
+
+* Ship the type fowarding facades directly in the `NETStandard.Library`
+  package.
+* Add a feature to MSBuild that allows resolving conflicts when multiple
+  packages provide a given assembly. This is outlined in more detail in the
+  section [MSBuild conflict resolution](#msbuild-conflict-resolution).
+
+### Which assemblies and packages does .NET Standard 2.0 need to be compatible with?
+
+As outlined in the [spec for .NET Standard 2.0][netstandard-spec] we need to be
+compatible with two major assembly sets:
+
+1. `mscorlib`-based assemblies
+2. `System.Runtime`-based assemblies
+
+These come from the following sources:
+
+1. Any assemblies that are part of .NET Framework and Portable Class Libraries
+   that contain types that are also in .NET Standard 2.0.
+2. All assemblies that are part of `NETStandard.Library` 1.x
+3. All assemblies that are in packages that only depend on `NETStandard.Library`
```

Comment:
It doesn't have to depend on "NETStandard.Library" the key is anything that used to ship in its own package that is no part of netstandard2.0.